src/openforms/registrations/contrib/zgw_apis/tests/test_utils.py

```diff
@@ -0,0 +1,75 @@
+from django.test import TestCase
+
+from ..utils import process_according_to_eigenschap_format
+
+
+class UtilsTests(TestCase):
+    def test_valid_date_is_successfully_processed(self):
+        specificatie = {
+            "groep": "",
+            "formaat": "datum",
+            "lengte": "14",
+            "kardinaliteit": "1",
+            "waardenverzameling": [],
+        }
+        sample = "2024-11-09"
+        processed_value = process_according_to_eigenschap_format(specificatie, sample)
+
+        self.assertEqual(processed_value, "20241109")
+
+    def test_valid_datetime_is_successfully_processed(self):
+        specificatie = {
+            "groep": "",
+            "formaat": "datum_tijd",
+            "lengte": "14",
+            "kardinaliteit": "1",
+            "waardenverzameling": [],
+        }
+        sample = "2024-01-31T12:07:00+01:00"
+        processed_value = process_according_to_eigenschap_format(specificatie, sample)
+
+        self.assertEqual(processed_value, "20240131130700")
```

Comment:
I don't think this is correct:

* `12:07:00` in UTC + 1 is `11:07:00` in UTC
* currently (in winter time), AMS is UTC+1, so the output would be (time part): `120700`

For reference, the following code should be correct:

```py
from openforms.contrib.zgw.clients.utils import TIMEZONE_AMS
from datetime import datetime

>>> sample = datetime.fromisoformat("2024-01-31T16:24:00+00:00")
>>> sample.astimezone(TIMEZONE_AMS).strftime("%Y%m%d%H%M%S")
'20240131172400'
```

but instead, it's easier to use Django's `django.utils.timezone.make_naive` function.

In summer, it's UTC+2, so the output would be one hour later for the same sample.